react/lib/util/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const FIAT_CURRENCIES = ['USD', 'CAD', 'EUR', 'GBP', 'AUD'];
```

Comment:
I'm pretty sure we don't support EUR, GBP and AUD yet right?

Reply:
@chedieck can confirm, he added this extra currencies  in the previous branch here : commit [8aa0c3fc7c5930ae1b76e8f95df4403b72b8729c](https://github.com/PayButton/paybutton/pull/403/commits/8aa0c3fc7c5930ae1b76e8f95df4403b72b8729c). I did a cherry pick in this commit and added to this branch

Reply:
this is just replicating the logic that we have on master now,
this constant was just like this on master: 
 
`export const fiatCurrencies = ['USD', 'CAD', 'EUR', 'GBP', 'AUD'] as const;` 

and it is used to verify if it is fiat or not with the function isFiat  and also used to check if the currency is supported in the isValidCurrency function,  so we need to fix the logic

Reply:
Ok well then let's create a new MR branched off this one to reduce the fait currencies down to just CAD and USD as that's all we currently support.